Documentation/ReleasePlan.md

```diff
@@ -0,0 +1,43 @@
+# Release Plan
+
+## Versioning strategy
+
+Coverlet is versioned with Semantic Versioning [2.0.0](https://semver.org/#semantic-versioning-200) that states:
+
+```
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+MAJOR version when you make incompatible API changes,
+MINOR version when you add functionality in a backwards-compatible manner, and
+PATCH version when you make backwards-compatible bug fixes.
+Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+```
+
+## Release Calendar
+
+We release 3 components as nuget packages:  
+
+**coverlet.msbuild.nupkg**  
+**coverlet.console.nupkg**  
+**coverlet.collector.nupkg**  
+
+We plan 1 release every [Q1](https://en.wikipedia.org/wiki/Calendar_year) if there is *at least* 1 new commit of source code on master.  
+**We release intermediate packages in case of severe bug or to unblock users.**
+
+### Current versions
+
+| Package        | **coverlet.msbuild** |
+| :-------------: |:-------------:|
+|**coverlet.msbuild**      | 2.6.1  |  
+|**coverlet.console**      | 1.5.1  |
+|**coverlet.collector**      | not yet released |  
+
+### Proposed next versions  
+
+We bump version based on Semantic Versioning 2.0.0 spec(PATCH is handled by Nerdbank.GitVersioning).  
+If we add fetures to **coverlet.msbuild.nupkg** we bump MINOR version for other two packages.
+
+
+| Release Date        | **coverlet.msbuild**           | **coverlet.console**  | **coverlet.collector** | **notes** |
+| :-------------: |:-------------:|:-------------:|:-------------:|:-------------:|
+| 1 July 2019      | 2.6 | 1.5 |   1.0 |               |
```

Comment:
We can skip June because we released less than month ago and likely we release collector asap.

Reply:
On notes we can note important PR that bump versions or fix important things.

Reply:
Are you planning to release a beta package now and a stable version in July?

Reply:
TBD could be an idea...and after that release start with quarter scheduling(1 september)

Reply:
Just throwing my thoughts. I am just a noob in release planning.

I think we can release beta only when we bump a version (major changes). This gives everyone time to adjust and if needed to verify, they can use beta. Otherwise, our nightly build plan would suffice I think.

Reply:
> I think we can release beta only when we bump a version (major changes)

The idea of nightly was for this reason...until today I see that users ask for a feature we implement and they clone repo and uses local package build to test, so the update will be used by few.
Today with nightly after merge we allow to test to interested user without pollute public nuget with too much intermedie package, we're a few dev and testing today is not "perfect"(IMO) and likely we would fix and release frequently. BTW we'll see along the way.